### PR TITLE
Add intake-elasticsearch

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

File: recipes/intake-elasticsearch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  run:
+    - python >=3.5
+    - intake >=0.3
+    - dask >= 1.0.0
+    - requests
+    - pandas
+
+test:
+  imports:
+    - intake_e;asticsearch
```
Comment (beckermr) [inline recipes/intake-elasticsearch/meta.yaml]: This looks like a typo?

File: recipes/intake-elasticsearch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - jinja2
+  run:
+    - python >=3.5
+    - intake >=0.3
+    - dask >= 1.0.0
```
Comment (beckermr) [inline recipes/intake-elasticsearch/meta.yaml]: I'd remove the space here between `>=` and the version number.

**Comment by martindurant** [comment]
Thanks for spotting that

**Comment by martindurant** [comment]
Failure for the one azure build appears to be a conda build or anvil bug? The other two builds succeed.

**Comment by martindurant** [comment]
ping @beckermr

**Comment by beckermr** [comment]
Woops! Sorry for the delay here. I checked circle by hand and it passed. LGTM!

**Comment by martindurant** [comment]
Thanks!

**Pull request merged**